items.json

```diff
@@ -90775,6 +91663,146 @@
     },
     "weapons": []
   },
+  {
+    "id": "crpg_kilij_h0",
+    "baseId": "crpg_kilij",
+    "name": "Kilij",
+    "culture": "Aserai",
+    "type": "OneHandedWeapon",
+    "price": 12893,
+    "weight": 1.77,
+    "tier": 13.9751415,
```

Comment:
Tier 13.9 - Please fix and bring down to tier match (presumably around 7.0 - 7.5 tier judging by higher tiers)

Reply:
Tier has been adjusted (H0 blade weight wasn't in line)

Reply:
Couldn't hit T7.5 without messing with pommel/guard/handle also, T7.6 still inline though imo